src/scripts/pbs_submit.sh

```diff
@@ -132,7 +135,7 @@ bls_set_up_local_and_extra_args
 [ -z "$bls_opt_queue" ] || grep -q "^#PBS -q" $bls_tmp_file || echo "#PBS -q $bls_opt_queue" >> $bls_tmp_file
 
 # Extended support for MPI attributes
-if [ "x$bls_opt_wholenodes" == "xyes" ] ; then
+if [ "x$bls_opt_wholenodes" == "xyes" ] && [ "$is_pbs_pro" -neq "0" ]; then
```

Comment:
You've got to do it within a select statement and apparently 'nodes' is deprecated in PBS Pro 12, which is the version Clemson is using.